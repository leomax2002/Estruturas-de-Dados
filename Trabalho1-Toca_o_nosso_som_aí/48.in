add MunRa
add NoSleepTillBrooklyn
add MalandroEhMalandro
add YoungWildFree
list
play
next MunRa
list
ended
current
next MunRa
list
fight