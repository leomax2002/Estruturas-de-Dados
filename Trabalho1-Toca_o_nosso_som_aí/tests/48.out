MunRa,NoSleepTillBrooklyn,MalandroEhMalandro,YoungWildFree
MunRa,NoSleepTillBrooklyn,MalandroEhMalandro,YoungWildFree
NoSleepTillBrooklyn
NoSleepTillBrooklyn,MunRa,MalandroEhMalandro,YoungWildFree
Jedi Wagner, assuma o comando!